include/libtorrent/add_torrent_params.hpp

```diff
@@ -388,6 +388,8 @@ TORRENT_VERSION_NAMESPACE_3
 		error_code internal_resume_data_error;
 #endif // TORRENT_ABI_VERSION
 
+        // the ca certificate to use in case of a magnet link
```

Comment:
I forgot to mention that new fields to `add_torrent_params` is an ABI break, and can't be done in 2.0. It has to go into the `master` branch.

Also, this documentation need to be more thorough. Maybe with a reference to the field that may exist in a .torrent file.

Reply:
Done.